src/FM36Tool/Spike.FM36Tool.Application/Spike.FM36Tool.Application.csproj

```diff
@@ -13,13 +11,13 @@
     <PackageReference Include="ESFA.DC.JobContextManager" Version="0.2.11-beta0001" />
     <PackageReference Include="ESFA.DC.Queueing" Version="0.1.18-beta0001" />
     <PackageReference Include="ESFA.DC.Serialization.Json" Version="0.1.39-beta0001" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="5.0.0-preview.4.20251.6" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="5.0.0-preview.4.20251.6" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="5.2.0" />
```

Comment:
Had to upgrade this package so that the messages could be posted using the ESFA.DC.Queueing package and .net6.
However, this dependency is deprecated, so recommend that the ESFA.DC.Queueing library is updated to use Azure.Messaging.ServiceBus by the DC team